docs/common/DocsExample.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="docs-example">
+
+
+    <div class="code-toggle-button">
+      <KIconButton  class="code-toggle"
+      :icon="isCodeVisible ? 'chevronUp' : 'codeToggle'"
+      tooltip="Toggle code visibility"
+      @click="toggleCodeVisibility"
+    />
+        </div>
+
+        <!-- Toggle code  -->
+    <div v-show="isCodeVisible" class="code-container">
+      <slot name="code" />
+    </div>
+
+    <!--  The component -->
+    <div class="example-container">
+      <slot />
+    </div>
+
+
+  </div>
+</template>
+
+
+<script>
+  import KIconButton from "../../lib/buttons-and-links/KIconButton.vue";
```

Comment:
I believe there's no need to import `KIconButton` for it to work since it's globally registered. Could you try remove please and see if it still works?